src/bolt/utils.ts

```diff
@@ -59,55 +63,122 @@ export async function getReactionData (event: ReactionAddedEvent | ReactionRemov
+        if (!pullRequestLink.length && message.attachments?.length) {
+            const {title} = message.attachments[0];
+            if (title) {
+                const match = /https:\/\/github.com\/[^ ]+?\/pull\/\d+/.exec(title);
+                pullRequestLink = match ? match[0] : '';
+            }
+        }
+
         return {
             slackMsgText: message.text,
             slackMsgId: message.client_msg_id,
             slackMsgUserId: message.user,
-            slackThreadTs: message.thread_ts ?? message.ts
+            slackThreadTs: message.thread_ts ?? message.ts,
+            pullRequestLink,
         } as ReactionData;
     };
     const botUserId = await getBotUserId();
-    let messageInfo = null;
+    let messageInfo = getMessageInfo(result);
+
+    if (!result.client_msg_id && botUserId === event.item_user && !messageInfo.pullRequestLink.length &&
+        messageInfo.slackMsgTs !== messageInfo.slackThreadTs) {
 
-    if (!result.client_msg_id && botUserId === event.item_user) {
-        messageInfo = getMessageInfo(result);
         result = await slackBotApp.client.conversations.replies({
             channel: event.item.channel,
             inclusive: true,
             latest: messageInfo.slackThreadTs,
             limit: 1,
             ts: messageInfo.slackThreadTs,
         });
-
+        messageInfo = getMessageInfo(result);
     }
 
-    messageInfo = getMessageInfo(result);
-
     result = await slackBotApp.client.chat.getPermalink({
         channel: event.item.channel,
         message_ts: messageInfo.slackThreadTs,
     });
 
     const {permalink} = result;
 
-    const match = /https:\/\/github.com\/[^ ]+?\/pull\/\d+/.exec(messageInfo.slackMsgText as string);
     return {
         ...messageInfo,
         slackPermalink: permalink as string,
         reaction: event.reaction,
         reactionUserId: event.user,
-        pullRequestLink: match ? match[0] : '',
         slackChannelId: event.item.channel,
     };
 }
 
+
+export async function postSlackMessage (slackMessage: ChatPostMessageArguments): Promise<void> {
+    await slackBotApp.client.chat.postMessage(slackMessage);
+}
+
+export async function sendCodeReviewSummary (channel: string): Promise<void> {
+    let text = '*Reminders!*';
+    let pendingCount: number = 0;
+    let inProgressCount: number = 0;
+
+    const result = await prisma.codeReview.groupBy({
+        by: ['status'],
+        where: {
+            status: {
+                in: ['pending', 'inprogress'],
+            },
+            slackChannelId: channel,
+        },
+        _count: {
+            status: true,
+        }
+    });
+
+    result.forEach((item) => {
+        switch (item.status) {
+            case 'pending':
+                pendingCount += item._count.status;
+                break;
+            case 'inprogress':
+                inProgressCount += item._count.status;
+        }
+    });
+
+    if (!pendingCount && !inProgressCount) {
```

Comment:
I assume. if (pendingCount == 0 &&  inProgressCount == 0) { - as we defined them on line 130...131

Reply:
That is correct, pendingCount + inProgressCount === 0

Reply:
I meant that !pendingCount comparison is the same as pendingCount  != undefined || pendingCount != null.
But they are defined so properly to compare them with 0.